src/components/ResourcesSection/index.css

```diff
@@ -0,0 +1,22 @@
+@import url("https://fonts.googleapis.com/css2?family=Salsa&display=swap");
```

Comment:
your teammates are importing this elsewhere I think. @tmansuroglu can you confirm

Reply:
it is imported inside index.html